cfme/configure/tasks.py

```diff
@@ -49,19 +103,62 @@ def is_task_finished(destination, task_name, expected_status, clear_tasks_after_
     expected_status = re.compile(expected_status, re.IGNORECASE)
 
     try:
-        row = tab_view.table.row(task_name=task_name, state=expected_status)
+        rows = list(tab_view.table.rows(task_name=task_name, state=expected_status))
     except IndexError:
         logger.warn('IndexError exception suppressed when searching for task row, no match found.')
         return False
 
+    # check state = finished for all tasks
+    return expected_num_of_tasks == len(rows), len(rows)
+
+
+def are_all_tasks_analysis_pass(name, expected_num_of_tasks, task_type='container',
```

Comment:
**Required**

This is a method in a common module, and should not default to any task type.

Reply:
Removed the default value for task_type